src/main/java/org/broadinstitute/hellbender/dev/tools/walkers/bqsr/BaseRecalibratorUprooted.java

```diff
@@ -522,6 +522,7 @@ private void generateReport() {
      */
```

Comment:
Rename `BaseRecalibratorUprooted` to something more sensible (perhaps something like `BaseRecalibratorWorker`?)